android/app/src/main/java/smart/delirover/app/MailmanTraveledDistanceActivity.java

```diff
@@ -0,0 +1,234 @@
+package smart.delirover.app;
+
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.Color;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.Toast;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.delirover.app.R;
+
+import org.eclipse.paho.client.mqttv3.IMqttActionListener;
+import org.eclipse.paho.client.mqttv3.IMqttDeliveryToken;
+import org.eclipse.paho.client.mqttv3.IMqttToken;
+import org.eclipse.paho.client.mqttv3.MqttCallback;
+import org.eclipse.paho.client.mqttv3.MqttMessage;
+//MQTT connection part of the code is built on https://github.com/DIT112-V21/smartcar-mqtt-controller.git

```

Comment:
Repeating my comment: Reuse existing code. Duplicating this much code is **wrong**.